libc/test/src/__support/HashTable/CMakeLists.txt

```diff
@@ -30,7 +29,7 @@ if (CPU_SUPPORTS_SSE2)
       group_test.cpp
```

Comment:
With the `FLAGS` option setup, you only need to create 1 test, without checking for `CPU_SUPPORT_SSE2`:
```
add_libc_test(
    group_test
    UNIT_TEST_ONLY
    SUITE
      libc-support-tests
    SRCS
      group_test.cpp
    DEPENDS
      libc.src.__support.HashTable.bitmask
      libc.src.stdlib.rand
      libc.src.string.memcmp
    DEPENDS
      libc.src.__support.HashTable.bitmask
      libc.src.stdlib.rand
      libc.src.string.memcmp
)
```
Then you after `cmake` command, you can check if the following targets are available for `ninja`:
```
  libc.test.src.__support.HashTable.group_test.__unit__
  libc.test.src.__support.HashTable.group_test.__unit__.__NO_DISABLE_SSE2_OPT
```
And check if the generated codes in `build/projects/libc/test/src/__support/HashTable/` match your expectation.

Thanks,

Reply:
I see. I forgot to remove that part.

Reply:
Do you mind adding a `TODO` at the test for me to investigating this?  I'm in the middle of re-implementing how the flags are controlled, and this will be a good use-case / example for that.

I think the PR is good to go, modulo this issue that I will investigate after merging.

Thanks,